wscript

```diff
@@ -67,6 +67,11 @@ def options(opt):
         default=False,
         help='Configure as debug variant.')
 
+    g.add_option('--disable-gccdeps',
+        action='store_true',
+        default=False,
+        help='Disable the use of GCC dependencies output method and use waf default method.')
```

Comment:
It's not needed, otherwise it wouldn't be an option 🙂 Anyone changing branches, and especially going back to earlier versions, is likely to encounter an error where a dependency file is missing. Only solution is to clear the build folder. This never happened before the gccdeps was introduced.

I'm also not confident on it since I've had a build do strange things on a real board, which after a clean build did go away - I have no way to prove it was the cause though.